pkg/webhook/immutable_fields_validator.go

```diff
@@ -380,6 +382,21 @@ func validateImmutableFieldsForLoggingLogMetricResource(oldSpec, spec map[string
 	return allowedResponse
 }
 
+func validateImmutableFieldsForSQLInstanceResource(oldSpec, spec map[string]interface{}) admission.Response {
+	ImmutableFields := []string{"cloneSource"}
+	var res []string
+	for _, field := range ImmutableFields {
+		if isImmutableFieldModified(oldSpec, spec, field) {
```

Comment:
I think we want to achieve the immutability check via CEL rules? Not a blocker though.

Reply:
fixed